package.json

```diff
@@ -2,7 +2,6 @@
   "name": "find-your-house-app",
   "version": "0.1.0",
   "private": true,
-  "proxy": "http://localhost:3000",
```

Comment:
Not necessary if you fix your cors problems. If you add the `withCredentials` option to an axios post request, you cannot use the wildcard accept on your cors policy in your rails app